### DISCUSSION TITLE
Setup integrazione

### DISCUSSION THREAD

**Discussion by Gam3sItaly** [aturri/ha-zcsazzurro#10] [Q&A]
Ciao Andrea, volevo chiederti come dovrei impostare il config con i dati di auth key e client code perché non ho ben capito come vadano inseriti. Io ho già chiesto tutti i dati all'Azzurro però mettendo nel config esattamente le righe che hai messo tu nella guida all'istallazione mi da errore.
Grazie e buona giornata

**Comment by aturri** [comment]
Ciao @Gam3sItaly,
ZCS dovrebbe mandarti un PDF dove è indicato una chiave da usare come header di autenticazione che inizia con "Zcs " + un codice client che mi avevano scritto per mail invece.

```
zcsazzurro:
  auth_key: 'Zcs xxxx'
  client_code: 'yyyy'
````

**Reply by Gam3sItaly** [reply]
Si si, mi sono accorto di aver sbagliato a mettere le credenziali nel config e sistemandole funziona perfettamente. 
Grazie per la risposta e per aver creato l'integrazione

**Reply by mbisax** [reply]
Salve a tutti, mi stò dedicando alla stessa integrazione ma, dopo già 2 email inviate a 2 differenti indirizzi, non ho ricevuto risposta da ZCS. 
Potreste, gentilmente, passarmi la email che avete usato per contattarli?
Ringrazio di cuore.

**Reply by lucagas79** [reply]
ciao Zcs  risponde molto velocemente senza problemi , va inviato Nome e cognome e il seriale dell'inverter.

**Reply by mbisax** [reply]
Ciao lucagas79, quello che mi manca è la email a cui spedire la richiesta poichè, i 2 indirizzi emails, a cui ho spedito la richiesta, non hanno risposto.
Ti ricordi, per caso, la email che usasti per ottenere risposta?
Grazie

**Reply by lucagas79** [reply]
Ciao , io ho inviato la richiesta dal form del sito. La risposta l’ho ricevuta da: service.azzurrozcs@zcscompany.com

saluti

**Reply by mbisax** [reply]
Grazie moltissime lucagas79.
Ho appena inviato una email all'indirizzo che mi hai, gentilmente, specificato.
Ora attendiamo la risposta.
